api/src/queries/project/project-view-queries.ts

```diff
@@ -3,6 +3,62 @@ import { getLogger } from '../../utils/logger';
 
 const defaultLog = getLogger('queries/project/project-view-queries');
 
+/**
+ * SQL query to get a single public (published) project.
+ *
+ * @param {number} projectId
+ * @returns {SQLStatement} sql query object
+ */
+export const getPublicProjectSQL = (projectId: number): SQLStatement | null => {
+  defaultLog.debug({ label: 'getPublicProjectSQL', message: 'params', projectId });
+
+  if (!projectId) {
+    return null;
+  }
+
+  const sqlStatement = SQL`
+    SELECT
+      project.project_id as id,
+      project_type.name as type,
+      project.name,
+      project.objectives,
+      project.location_description,
+      project.start_date,
+      project.end_date,
+      project.caveats,
+      project.comments,
+      project.coordinator_first_name,
+      project.coordinator_last_name,
+      project.coordinator_email_address,
+      project.coordinator_agency_name,
+      project.coordinator_public,
+      project.geojson as geometry,
+      project.create_date,
+      project.create_user,
+      project.update_date,
+      project.update_user,
+      project.revision_count,
```

Comment:
I would remove these.  partially because we don't use them on the frontend (right?), and partially because it contains internal information about the project that probably shouldn't be public (mainly the user info).